packages/gestalt-design-tokens/tokens/color/alias.json

```diff
@@ -3,52 +3,64 @@
     "text": {
       "default": {
         "value": "{color.black.cosmicore.900.value}",
+        "darkValue": "{color.gray.roboflow.50.value}",
```

Comment:
Worth adding a comments that color values change between light/dark mode?

Reply:
I can add a comment to the top of the file, or did you mean per token?